public/js/components/SourceFooter.js

```diff
@@ -43,6 +44,10 @@ const SourceFooter = React.createClass({
     const { selectedSource, sourceText, togglePrettyPrint } = this.props;
     const sourceLoaded = selectedSource && !sourceText.get("loading");
 
+    if (isGenerated(selectedSource.toJS())) {
```

Comment:
Don't we want to hide it for sources that have sourcemaps *and* original sources?